cabot/metricsapp/api/elastic.py

```diff
@@ -1,11 +1,54 @@
 from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+class SupportedMetrics(Enum):
```

Comment:
Would request more doc strings on this class - to understand what this means. What is the difference between single and multiple? How do we decide on one v/s another? - based on `value` v/s `values` alone?